### ISSUE TITLE
Max vram clock not going past 1075MHz on 6700xt

### ISSUE THREAD

**Issue by notimadam123** [ilya-zlobintsev/LACT#363]
### Checklist

- [X] The problem is not listed in the [hardware support matrix](https://github.com/ilya-zlobintsev/LACT?tab=readme-ov-file#hardware-support) as a known limitation
- [X] I've included a [debug snapshot](https://github.com/ilya-zlobintsev/LACT?tab=readme-ov-file#reporting-issues) if the issue is related to hardware configuration

### Bug description

Max vram clock wont go past 1075mhz on a sapphire pulse 6700xt

### System info

```markdown
- LACT version:0.5.5-1 (aur)
- GPU model:Sapphire Pulse 6700xt
- Kernel version:6.10.6-zen1-1-zen
- Distribution: Arch Linux
```

**Comment by ilya-zlobintsev** [comment]
This is normal for a 6700XT, Linux shows the physical memory clock instead of the effective one, so the value will be half of the one listed in the specs or on windows.

This should probably be indicated in the UI to avoid confusion given that this is asked somewhat regularly.

**Comment by shanefagan** [comment]
Similar issue for me on a 7900xtx liquid devil, max VRAM clock is significantly lower than the 2700 ish max clock I get on Windows

**Comment by ilya-zlobintsev** [comment]
@shanefagan please read the previous comment

>  Linux shows the physical memory clock instead of the effective one, so the value will be half of the one listed in the specs or on windows.

The equivalent to 2700MHz on windows is shown as 1350 on linux

**Comment by shanefagan** [comment]
Ah interesting, yeah then it would be nice to have some note like you mentioned.

**Comment by ilya-zlobintsev** [comment]
I've decided that it's simpler to just always show the effective clock value in the UI. Now it will display the value that you'd normally expect.

Implemented in #368
